volunteers/templates/static/promo.html

```diff
@@ -3,38 +3,38 @@
 <div id="intro">
     <h2 class="intro"><span>FOSDEM</span><br/>Volunteering</h2>
 
-    <p class="intro">This is a site that brings all the volunteers together.
-    Find all the cool tasks that you are interested in.</p>
+    <p class="intro">This site brings all volunteers together.
+    Find cool tasks you are interested in.</p>
 
-    <p id="xtra-info">Problems? For bugs you can report an issue at
-    <a href="https://github.com/FOSDEM/volunteers/issues">Github</a>.</p>
+    <p id="xtra-info">Problems? Fix or report bugs as issues at
+    <a href="https://github.com/FOSDEM/volunteers/issues">GitHub</a>.</p>
 </div><!-- /intro-->
 
 <ul class="features">
     <li><h3>FOSDEM</h3>
-        <p><a href="https://www.fosdem.org">FOSDEM</a> is a free event that offers open source
+        <p><a href="https://www.fosdem.org">FOSDEM</a> is a gratis event that offers libre software
         communities a place to meet, share ideas and collaborate.</p>
     </li>
 
-    <li><h3>Location</h3>
-        <p>FOSDEM will take place at the<a href="https://fosdem.org/practical/transportation/">
+    <li><h3><a href="https://www.openstreetmap.org/way/30358555#map=15/50.8207/4.3966">Location</a></h3>
+        <p>FOSDEM takes place at the<a href="https://fosdem.org/practical/transportation/">
         ULB Solbosch Campus, Brussels, Belgium, Europe, Earth</a>.</p>
     </li>
 
     <li><h3>Volunteering</h3>
-        <p>FOSDEM needs you! Every year, the FOSDEM team is assisted by an enthusiastic team of volunteers
-        to help out with various tasks and make the event a fun and safe place for all our visitors.</p>
+        <p>FOSDEM needs you! Every year. Enthusiastic volunteers assist the FOSDEM team,
+        in making the event—fun and safe for all.</p>
```

Comment:
Typo? (dash can just be a space)

Reply:
It is an em dash, was going for the dual meaning,
_helping make the actual event_, and,
_helping make the event fun and safe_

Reply:
Ahhhhhh. That's best unpacked. Sth like "making the event happen and making it fun and safe for all"?

Reply:
It is no less fun for volunteers to help the FOSDEM team. Aha.